packages/workbox-background-sync/test/browser/background-sync-queue.js

```diff
@@ -57,60 +57,65 @@ describe('background sync queue test', () => {
     backgroundSyncQueue = new workbox.backgroundSync.test.BackgroundSyncQueue({
       maxRetentionTime: MAX_AGE,
       queueName: QUEUE_NAME,
       callbacks: CALLBACKS,
     });
-    chai.assert.isObject(backgroundSyncQueue._queue);
-    chai.assert.isObject(backgroundSyncQueue._requestManager);
-    chai.assert.equal(backgroundSyncQueue._queue._queueName, QUEUE_NAME);
-    chai.assert.equal(backgroundSyncQueue._queue._config.maxAge, MAX_AGE);
-    chai.assert.equal(backgroundSyncQueue._requestManager._globalCallbacks,
-      CALLBACKS);
+    expect(backgroundSyncQueue._queue).to.be.an('object');
+    expect(backgroundSyncQueue._requestManager).to.be.an('object');
+    expect(backgroundSyncQueue._queue._queueName).to.be.equal(QUEUE_NAME);
+    expect(backgroundSyncQueue._queue._config.maxAge).to.be.equal(MAX_AGE);
+    expect(backgroundSyncQueue._requestManager._globalCallbacks).to.be
+        .equal(CALLBACKS);
   });
 
-  it('check push proxy', async function() {
-    await backgroundSyncQueue.pushIntoQueue({request: new Request('/__echo/counter')});
-    chai.assert.equal(backgroundSyncQueue._queue.queue.length, 1);
+  it('should push request in queue via pushIntoQueue method', async function() {
+    await backgroundSyncQueue.pushIntoQueue(
+        {request: new Request('/__echo/counter')});
+    expect(backgroundSyncQueue._queue.queue.length).to.be.equal(1);
   });
 
-  it('check replay', async function() {
-    await backgroundSyncQueue.pushIntoQueue({request: new Request('/__echo/counter')});
-    await backgroundSyncQueue.pushIntoQueue({request: new Request('/__echo/counter')});
-    chai.assert.equal(backgroundSyncQueue._queue.queue.length, 2);
+  it('check replay queued request via replayRequests method', async function() {
+    await backgroundSyncQueue.pushIntoQueue(
+        {request: new Request('/__echo/counter')});
+    await backgroundSyncQueue.pushIntoQueue(
+        {request: new Request('/__echo/counter')});
+    expect(backgroundSyncQueue._queue.queue.length).to.be.equal(2);
     await backgroundSyncQueue.replayRequests();
-    chai.assert.equal(responseAchieved, 2);
+    expect(responseAchieved).to.be.equal(2);
   });
 
-  it('check replay failure with rejected promise', async function() {
-    await backgroundSyncQueue.pushIntoQueue({request: new Request('/__echo/counter')});
-    await backgroundSyncQueue.pushIntoQueue({request: new Request('/__test/404')});
+  it('should rejected promise on replay failure', async function() {
+    await backgroundSyncQueue.pushIntoQueue(
+        {request: new Request('/__echo/counter')});
+    await backgroundSyncQueue.pushIntoQueue(
+        {request: new Request('/__test/404')});
     try {
       await backgroundSyncQueue.replayRequests();
       throw new Error('Replay should have failed because of invalid URL');
     } catch (err) {
-      chai.assert.equal(404, err[0].status);
+      expect(404).to.be.equal(err[0].status);
```

Comment:
`expect` comparisons should compare the expected result with the actual result, so this should be reversed.

```js
expect(err[0].status).to.equal(404);
```